src/read_write/lg_interface.jl

```diff
@@ -0,0 +1,48 @@
+LightGraphs.nv(g::AbstractGraph{V,E}) where {V,E} = num_nodes(g)
+LightGraphs.ne(g::AbstractGraph{V,E}) where {V,E} = num_edges(g)
+
+function LightGraphs.outneighbors(g::AbstractStaticGraph{NT,E}, v::V) where {V, NT<:AbstractNode{V},E}
+    edges = out_edges(g, v)
+    return Node{V}[target(e) for e in edges]
+end
+
+function LightGraphs.inneighbors(g::AbstractStaticGraph{NT,E}, v::V) where {V, NT<:AbstractNode{V},E}
+    edges = in_edges(g, v)
+    return Node{V}[source(e) for e in edges]
+end
+
+LightGraphs.src(e::AbstractEdge{V}) where V = source(e)
+LightGraphs.dst(e::AbstractEdge{V}) where V = target(e)
+
+LightGraphs.has_edge(g::AbstractGraph{V,E}, e::AbstractEdge{V}) where {V,E} = has_edge(g,e)
+LightGraphs.has_vertex(g::AbstractGraph{V,E}, n::AbstractNode{V}) where {V,E} = has_node(g,n)
+
+Base.reverse(e::AbstractEdge{V}) where V = edge_reverse(e)
+
+LightGraphs.edges(g::AbstractGraph{V,E}) where {V,E} = edges(g)
+LightGraphs.edgetype(::AbstractGraph{V,E}) where {V,E} = E
+
+LightGraphs.vertices(g::AbstractGraph{V,E}) where {V,E} = nodes(g)
+
+LightGraphs.is_directed(g::AbstractGraph{V,E}) where {V,E} = is_directed(g)
+
+LightGraphs.add_edge!(g::AbstractStaticGraph{V}, e::AbstractEdge{ET}) where {V,ET} = add_edge!(g, LightGraphs.src(e), LightGraphs.dst(e))
```

Comment:
I think this should be 
```
LightGraphs.add_edge!(g::AbstractStaticGraph{V,E}, e::AbstractEdge{V})
```

Reply:
I added the two, to be able to define an edge by keys and not indices
```
LightGraphs.add_edge!(g::AbstractStaticGraph{V,E}, e::AbstractEdge{V}) where {V,E} = add_edge!(g, LightGraphs.src(e), LightGraphs.dst(e))
LightGraphs.add_edge!(g::AbstractStaticGraph{V,E}, e::AbstractEdge{T}) where {T, V<:AbstractNode{T},E} = add_edge!(g, LightGraphs.src(e), LightGraphs.dst(e))
```